src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -532,6 +536,9 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         auto pfnScrollPositionChanged = std::bind(&TermControl::_TerminalScrollPositionChanged, this, std::placeholders::_1, std::placeholders::_2, std::placeholders::_3);
         _terminal->SetScrollPositionChangedCallback(pfnScrollPositionChanged);
 
+        _autoScrollTimer.Interval(std::chrono::microseconds(static_cast<int>(1.0 / 30.0 * 1000000)));
+        _autoScrollTimer.Tick({ this, &TermControl::_UpdateAutoScroll });

```

Comment:
I believe that this timer also needs to get torn down cleanly in `TermControl::Close`, like `_cursorTimer`